pipeline/trusted-resources/pkg/trustedtask/trustedtask.go

```diff
@@ -0,0 +1,82 @@
+package trustedtask
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	"github.com/tektoncd/pipeline/pkg/client/injection/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/pkg/apis"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	"knative.dev/pkg/logging"
+)
+
+type TrustedTask struct {
+	v1beta1.Task
+}
+
+func (t *TrustedTask) Validate(ctx context.Context) (errs *apis.FieldError) {
+	k8sClient := kubeclient.Get(ctx)
+	tektonClient := client.Get(ctx)
+
+	if err := t.verifyTask(ctx, k8sClient, tektonClient); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (t *TrustedTask) verifyTask(
+	ctx context.Context,
+	k8sClient kubernetes.Interface,
+	tektonClient versioned.Interface) (errs *apis.FieldError) {
+	logger := logging.FromContext(ctx)
+	logger.Info("Verifying Task")
+
+	task := t.Task
+
+	cfg := config.FromContextOrDefaults(ctx)
+	cfg.FeatureFlags.EnableTektonOCIBundles = true
+	ctx = config.ToContext(ctx, cfg)
```

Comment:
Ideally it'd be great if we didn't need to inject this into the context, but if this is always going to be set for every type, consider setting this for the whole controller here - https://github.com/tektoncd/experimental/blob/b94e00f1de513635d831c4ce239cd7c8f171ac60/pipeline/trusted-resources/cmd/webhook_trusted_resource/main.go#L65

Reply:
didn't realize this. thanks this hint! 

mark this as a follow up action. see if need certain flexibility or not here for following work. if always, will update in webhook. 
cc @Yongxuanzhang